site/docs/v1/tech/admin-guide/licensing.adoc

```diff
@@ -91,6 +91,24 @@ Should you need to deactivate your license Id, either because you are changing y
 
 image::admin-guide/license-activation/reactor-activated-decommission.png[Decomission Reactor.,width=1200]
 
+Deactivating your license will disable any link:/docs/v1/tech/premium-features/[premium functionality].
+
+This includes both the ability of users to access such functionality and the ability of admins to configure it.
+
+For example, consider the scenario where you:
+
+* activate your license
+* enable email MFA on your tenant
+* have users set up email MFA on their accounts
+
+Then, after some time, you deactivate your license. While your license is deactivated, the following functionality will be affected
+
+* users will not be able to configure email MFA
```

Comment:
Capitalize first word and periods. I think these are all sentences. Unordered list is good here because they are all side effects.